src/main/java/org/launchcode/liftoffproject/models/Child.java

```diff
@@ -88,16 +86,21 @@ public void setUserAccount(ChildUser userAccount) {
         this.userAccount = userAccount;
     }
 
-    public List<Chore> getChore() {
-        return chore;
+
+    public int getEarnedPoints() {
+        return earnedPoints;
     }
 
-    public void setChore(List<Chore> chore) {
-        this.chore = chore;
```

Comment:
Think we need the getters/setters for Chore because of the way Thymeleaf/Hibernate works